### ISSUE TITLE
Right motor not moving backwards

### ISSUE THREAD

**Issue by Kheiden** [Kheiden/RPi-tankbot#86]
The wires on the breadboard were accidentally taken out and put back incorrectly.

After re-wiring the breadboard, I noticed that the right motor moves forward along with the left motor when initiating the `forward` command, however the right motor does not move backwards when initiating the `backwards` command.

I think the L293D motor controller may be broken, so I've ordered another one and will attempt the movement again after swapping the controller.

If replacing the motor controller does not work, I'll need to change the GPIO pins on the RPI as they may be broken.

**Comment by Kheiden** [comment]
After swapping the motor controller, the tank was able to perform the movements listed above but not backwards. This indicates to me that the motor controller was probably not the issue and the issue lies either in the breadboard or the GPIO pins.

**Comment by Kheiden** [comment]
I manually added 5V to the right motor and it moved backwards.

**Comment by Kheiden** [comment]
I believe the issue lies with `Motor2B,GPIO.LOW`, however I switched the GPIO port did not fix the issue. This could mean that the breadboard is the issue.

**Comment by Kheiden** [comment]
I rewired the pins to a new part of the breadboard and it still doesn't work.

**Comment by Kheiden** [comment]
I just realized that although I switched the pins in [this ](https://github.com/Kheiden/RPi-tankbot/issues/86#issuecomment-450534338)comment and changed the code, I forgot to deploy the code to the RPI.

Todo: rewire the pin for `Motor2B,GPIO.LOW` then change AND PUSH the code.

**Comment by Kheiden** [comment]
I fully charged all 4 batteries (according to the battery charger). The voltages for each battery is below:
Battery 0: 4.21V
Battery 1: 4.21V
Battery 2: 4.21V
Battery 3: 4.21V

I turned on the power switch for the Robot and was able to successfully ssh into the RPi. I started the local webserver then successfully loaded the Web App Controller. I then configured the local webserver as a systemd daemon and enabled it to run on boot. I rebooted the RPi and reloaded the web page roughly two times per second and the page was able to successfully load after about two minutes.

The Camera Stream works fine for both left and right cameras.

The robot can move forward, but the right motor can not move in reverse. The left motor can move backwards when the robot is being issues a "backwards" command via the API.

There's an apparent hardware limitation that is preventing the robot's right motor from moving backwards. Because the robot is mobile, it should be possible to navigate the room without having to use the right backwards functionality.

I'll de-prioritize this issue to work on the software aspect.
